src/platform/forms/constants.js

```diff
@@ -276,3 +276,270 @@ export const SIP_ENABLED_FORMS = new Set([
   VA_FORM_IDS.FORM_VA_2346A,
   VA_FORM_IDS.FORM_5655,
 ]);
+
+export const MY_VA_SIP_FORMS = [
+  {
+    id: VA_FORM_IDS.FORM_10_10EZ,
+    benefit: 'health care benefits',
+    title: 'health care benefits (10-10EZ)',
+    description: 'health care benefits application (10-10EZ)',
+    link: '/health-care/apply/application/',
+    trackingPrefix: 'hca-',
+    isSipEnabled: true,
```

Comment:
I didn't see any usage of this property `isSipEnabled`. Is this something that is going to be used later on in some way? All the objects in this array are set to true for this, so I kind of assumed that the array itself would only ever really contain SIP enabled forms, so maybe this could be removed to cut down on noise?

Also, I noticed that this first object has a `link` property but none of the other objects have this property. Not sure if that is purposeful or not.